bootstrap_cfn/config.py

```diff
@@ -90,6 +90,11 @@ def process(self):
         if 's3' in self.data:
             s3 = self.s3()
             map(template.add_resource, s3)
+            template.add_output(Output(
+                "StaticBucketName",
+                Description="S3 bucket name",
+                Value=Ref("StaticBucket")
+            ))
```

Comment:
I've used the 'returning the output list' option with ELB but I'm moving more towards the 'passing the template' option